packages/admin-theme/src/typographyOptions.ts

```diff
@@ -44,8 +44,8 @@ export const typographyOptions: TypographyOptions = {
     h6: {
         fontFamily,
         fontSize: 16,
-        lineHeight: 1.42,
-        fontWeight: fontWeights.fontWeightBold,
+        lineHeight: "19px",
```

Comment:
Shouldn't the line height be `normal`?

![image](https://user-images.githubusercontent.com/48853629/134499197-34f53d3d-6d7c-4a23-b7c4-42b241df361f.png)

https://zeroheight.com/5d7649959/p/80e845-typography/b/01fc09/t/591b0b

Reply:
I'm not sure 🤔 
According to zeroheight: `font-weight: 500; line-height: normal;`
According to invision: `font-weight: 400; line-height: 19px|20px;`

Reply:
Okay, let's keep the Invision values, as they're more up to date

Reply:
@johnnyomair I spoke to Mitch, the Zeroheight values have now been updated and I used those values here.

Reply:
"normal" never saw that before :)

Reply:
"normal" is the default-value but I also never saw it before 😅